pkgs/development/compilers/mosml/default.nix

```diff
@@ -15,6 +15,8 @@ stdenv.mkDerivation rec {
 
   setSourceRoot = ''export sourceRoot="$(echo */src)"'';
 
+  dontPatchELF = true;
```

Comment:
Please add a comment why this is needed.

Reply:
Comment added.